packages/web-sdk/src/transports/otlp/payload/transform/transform.ts

```diff
@@ -197,3 +182,45 @@ export function initLogsTransform(internalLogger: InternalLogger): LogsTransform
     toLogRecord,
   };
 }
+
+export function getTraceTransforms(_internalLogger?: InternalLogger): TraceTransform {
+  function toResourceSpan(transportItem: TransportItem<TraceEvent>): ResourceSpans {
+    const resource = toResource(transportItem);
+    const scopeSpans = transportItem.payload.resourceSpans?.[0]?.scopeSpans;
+
+    return {
+      resource,
+      scopeSpans: scopeSpans ?? [],
+    };
+  }
+
+  return {
+    toResourceSpan,
+  };
+}
+
+function toResource(transportItem: TransportItem): Readonly<Resource> {
+  const { browser, sdk, app }: ResourceMeta = transportItem.meta;
+
+  return {
+    attributes: [
+      toAttribute(SemanticBrowserAttributes.BROWSER_MOBILE, browser?.mobile),
+      toAttribute(SemanticBrowserAttributes.BROWSER_USER_AGENT, browser?.userAgent),
+      toAttribute(SemanticBrowserAttributes.BROWSER_LANGUAGE, browser?.language),
+      toAttribute('browser.os', browser?.os),
+      // toAttribute(SemanticBrowserAttributes.BROWSER_BRANDS, browser?.brands),
```

Comment:
Any reason to leave this in? Future purposes?

Reply:
THnaks Elliot, yes this is left on purpose because we need to decide how to add that data. 
Why?:
Otel defines browser name and version in to be in the same string.
Thosename+versioncombinations are added to the `browser.brands` array.
Loosely such a list represents a group of browsers believed to be compatible with each other.

This information is received from [`navigator.userAgentData.brands` which currently is only suppported in Chromium based browsers](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorUAData/brands#browser_compatibility).
For us it would be hard to decide what is compatible engine etc. 

So this will be done in an extra PR to get some more opinions in how to solve it.

Mr current idea is to discuss two things:

1. Detect if browser supports `NavigatorUAData`, then query it directly from that interface.
If not, then add browserName+version string to the brands list. 
This would of course leave out the additional items (compatible browsers) that Browsers which have support for the API do provide (see example below from Chrome). 

2. Simply wait till all browsers have support an keep our current Faro approach (it's how it is implemented atm)

3. A mix of the above. If a browser has support for the API also add browser.brands, if not leave it empty and keep the current model which is `browser.name: string` / `browser.version: string`